aiohttp_rpc/server/websocket.py

```diff
@@ -12,55 +13,51 @@
 
 
 class WsJsonRpcServer(BaseJsonRpcServer):
-    ws_state_key = 'rcp_websockets'
+    rcp_websockets: weakref.WeakSet
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.rcp_websockets = weakref.WeakSet()
 
     async def handle_http_request(self, http_request: web.Request) -> web.StreamResponse:
         if http_request.method != 'GET' or http_request.headers.get('upgrade', '').lower() != 'websocket':
             return web.HTTPMethodNotAllowed(method=http_request.method, allowed_methods=('POST',))
```

Comment:
I assume the allowed method here should be `GET` and that it should raise, not return `HTTPMethodNotAllowed` which is incompatible with `StreamResponse`.